pricing.js

```diff
@@ -1,36 +1,41 @@
+function volLife(product, price, dollarsOff) {
+  if (product.employerContribution.mode === 'dollar') {
+    price = price - product.employerContribution.contribution
+  } else {
+    dollarsOff = price * (product.employerContribution.contribution / 100)
+    price = price - dollarsOff
+  }
+  return price
+}
+function formatPrice(price) {
+  price = parseInt(price * 100) / 100
+  return price
+}
+function ltdCoverage(price, employee, product) {
+  var salaryPercentage = product.coveragePercentage / 100
+  price += ((employee.salary * salaryPercentage) / product.cost.costDivisor) * product.cost.price
+  return price
+}
+function ourCoverageAmount(price, coverageLevels, product) {
+  for (var i = 0; i < coverageLevels.length; i++) {
+    var coverageAmount = coverageLevels[i].coverage
+    price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+  }
+  return price
+}
 module.exports.calculateProductPrice = function (product, employee, coverageLevels) {
   var price = 0
   var dollarsOff = 0
 
   switch (product.type) {
     case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
-
-        price += (coverageAmount / product.cost.costDivisor) * product.cost.price
-      }
-
-      if (product.employerContribution.mode === 'dollar') {
-        price = price - product.employerContribution.contribution
-      } else {
-        dollarsOff = price * (product.employerContribution.contribution / 100)
-        price = price - dollarsOff
-      }
-
-      return parseInt(price * 100) / 100
+      price = ourCoverageAmount(price, coverageLevels, product)
+      price = volLife(product, price, dollarsOff)
+      return formatPrice(price)
     case 'ltd':
-      var salaryPercentage = product.coveragePercentage / 100
-
-      price += ((employee.salary * salaryPercentage) / product.cost.costDivisor) * product.cost.price
-
-      if (product.employerContribution.mode === 'dollar') {
-        price = price - product.employerContribution.contribution
-      } else {
-        dollarsOff = price * product.employerContribution.contribution
-        price = price - dollarsOff
-      }
-
-      return parseInt(price * 100) / 100
+      price = ltdCoverage(price, employee, product)
+      price = volLife(product, price, dollarsOff)
```

Comment:
See how that name `volLife` becomes confusing down here. We're working on an LTD product and calling a function named volLife which is the other product type.